staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -604,6 +638,21 @@ func (c *Config) Complete(informers informers.SharedInformerFactory) CompletedCo
 		}
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.APIServerIdentity) {
+		config := c.APIServerIDConfig
+		if config == nil {
+			inClusterConfig, err := restclient.InClusterConfig()
+			if err != nil {
+				klog.Fatalf("cannot determine config for API server lease management: %v", err)
+			}
+			config = inClusterConfig
+		}
+
+		config = restclient.CopyConfig(config)
+		// use protobuf here because we know the kube-apiserver supports it for leases
+		config.ContentConfig.ContentType = "application/vnd.kubernetes.protobuf"
+	}
```

Comment:
```suggestion
		config.ContentConfig.ContentType = "application/vnd.kubernetes.protobuf"
		c.APIServerIDConfig = config
	}
```

missing assignment?

Reply:
hhuurrg must have screwed up a rebase somewhere.

Reply:
Fixed.  Thanks for catching this :)